recipes-debian/ifenslave/ifenslave_debian.bb

```diff
@@ -0,0 +1,35 @@
+# base recipe: meta-openembedded/meta-networking/recipes-support/ifenslave/ifenslave_2.9.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+
+inherit debian-package
+require recipes-debian/sources/ifenslave.inc
+
+SUMMARY = "Configure network interfaces for parallel routing"
+HOMEPAGE = "http://www.linuxfoundation.org/collaborate/workgroups/networking/bonding"
+SECTION = "net"
+
+LICENSE = "GPLv3"
+LIC_FILES_CHKSUM = "file://debian/copyright;md5=acc89812938cf9ad6b1debc37cea0253"
+
+inherit manpages
+MAN_PKG = "${PN}"
+
+SRCREV = "42bfbb9beb924672ca86b86e9679ac3d6b87d992"
```

Comment:
Is this used?

Reply:
Let me confirm

Reply:
I confirmed that there was no problem, so I deleted it.